_maps/map_files/TgStation/tgstation.2.1.3.dmm

```diff
@@ -22358,6 +22358,12 @@
 	dir = 2;
 	icon_state = "pipe-c"
 	},
+/obj/machinery/button/door{
+	id = "datboidetective";
```

Comment:
IIIRC you can decon buttons to change the set tag, so people might see it if they're real dedicated to checking.